Gems/LyShine/Code/Source/UiParticleEmitterComponent.h

```diff
@@ -179,6 +181,14 @@ class UiParticleEmitterComponent
     void SetOverrideAlpha(float alpha) override;
     // ~UiVisualInterface
 
+    // UiIndexableImageBus

```

Comment:
Nit: Per coding standard should be:
```suggestion
    // UiIndexableImageBus overrides ...
```